README.md

```diff
@@ -11,6 +11,20 @@
 
 > An [ESLint](https://eslint.org/) rule that reports usage of deprecated code.
 
+Reports usage of anything marked with a `@deprecated` JSDoc tag (browser APIs, Node.js APIs, library APIs, etc)

```

Comment:
```suggestion
Reports usage of any deprecated code which is marked with a [`@deprecated` JSDoc tag](https://jsdoc.app/tags-deprecated.html).

```

Reply:
I think this removes a main point of value for this PR (clarifying the unclear point of "which code could this be?" by removing the 3 examples)

Reply:
I specifically do not want to mention any APIs whatsoever and added "any" to "deprecated code" phrase to substitute specifics, otherwise this may lead to more confusion, such as why specifically those APIs are mentioned, does that mean it will not work for other use-cases etc.
This rule is fully agnostic and does not care what kinds of APIs you are using or where the code is coming from, as long as it's JS/TS code which is marked with deprecated jsdoc it will report it's usage.

Reply:
I think you're misunderstanding the point of providing these examples, so let me try again:

People all over the place are not understanding what `eslint-plugin-deprecation` does. They don't know what type of code it could possibly apply to. Saying "any code" is technically, purely correct, but completely dismisses the difficulties of these people and lacks proper teaching (which is what docs should do)

Teaching can be easily done here by providing some simple examples of what types of code this plugin deals with. Examples are a pretty common teaching tool. These examples have already proved to be valuable by my responses on Twitter to users confused by the current docs (see #72).

Adding examples of what types of code could apply here (including "etc." or similar language) does not discount from the words "any code" in any way, and does not indicate that `eslint-plugin-deprecation` does not handle other types of code outside of the examples

Reply:
I updated "anything" to "any code" and added "incl."

```diff
-Reports usage of anything marked with a `@deprecated` JSDoc tag (browser APIs, Node.js APIs, library APIs, etc)
+Reports usage of any code marked with a `@deprecated` JSDoc tag (incl. browser APIs, Node.js APIs, library APIs, etc.)
```